components/compliance-service/migrations/upgrades.go

```diff
@@ -0,0 +1,62 @@
+package migrations
+
+import (
+	"context"
+	"fmt"
+	"github.com/chef/automate/components/compliance-service/dao/pgdb"
+	"github.com/chef/automate/lib/cereal"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+type Upgrades struct {
+	DB            *pgdb.DB
+	cerealManager *cereal.Manager
+}
+
+func New(db *pgdb.DB, cerealManager *cereal.Manager) *Upgrades {
+	return &Upgrades{db, cerealManager}
+}
+
+const getUpgradeValueForDayLatest = `
```

Comment:
declare the flag as constant, we should not declare the statements as constants. we need to write a common method that takes the flag and returns the query stmt.